app/controllers/contacts_controller.rb

```diff
@@ -4,16 +4,18 @@
 class ContactsController < ApplicationController
   skip_verify_authorized
 
-  def new
-    @contact_form = ContactForm.new
-  end
+  def new; end
 
   def create
     contact_form = ContactForm.new(contact_form_params)
 
     ContactsMailer.with(contact_form.attributes.symbolize_keys).jira_email.deliver_later
 
-    redirect_to success_contacts_path
+    redirect_to success_contacts_path(bounce_location: params.dig('contact', 'bounce_location'))
```

Comment:
Can you document what bounce location is doing?

Reply:
It allows the `Close` link on the "success" view to point at the app root path instead of as a modal dismiss. (Not planning to document that yet as I consider whether to refactor as a turbo-frame.)